UncomplicatedCustomRoles/Patch/RespawnTimerPatch.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.Emit;
+using Exiled.API.Features;
+using Exiled.API.Features.Roles;
+using HarmonyLib;
+using PlayerRoles;
+using Respawning;
+using RespawnTimer.API.Features;
```

Comment:
The whole file requires RespawnTimer as hard-dependency, we must use reflection

Reply:
This is why I used an external plugin to avoid symbols not defined in the main plugin.

Reply:
reflection does not causes that, see https://github.com/UncomplicatedCustomServer/UncomplicatedCustomRoles/blob/main/UncomplicatedCustomRoles/Manager/SpawnManager.cs#L166

Reply:
It is better to have a compilation error than a RunTime error. When we use reflection the compiler does not know if the method continues to exist

Reply:
NB: i will change it with relfxion. I had a meeting today. And this afternoon I am doing practical work. I should also correct and put mark on some projects soon. Just to say "yea i ave some work... I back soon to do the edit".